src/test/resources/org/jabref/logic/xmp/article_dublinCore_partial_date.bib

```diff
@@ -0,0 +1,8 @@
+@Article{,
+  coverage  = {coverageField},
+  language  = {English,Japanese},
+  month     = {#jan#},
```

Comment:
This is a very strange BibTeX. It should read:

```suggestion
  month     = jan,
```

See https://github.com/JabRef/jabref/blob/main/docs%2Fadr%2F0024-use-%23-as-indicator-for-BibTeX-string-constants.md for the documentation of the alternatives; and https://docs.jabref.org/setup/stringeditor for the user documentation.

In short: if `{#jan#}` appears in a `.bib` file, the file is wrong.

Reply:
Well, apparently if you use bibEntry.setMonth, that's the format you should expect as seen below
![image](https://user-images.githubusercontent.com/4951690/157482675-47d27f02-e071-48db-ad76-de6886bc329d.png)

Maybe I could change it to 
```setField(StandardField.MONTH, parsedMonth.getShortName()) ```
in extractDate function